Api/SplinterlandsAPI.cs

```diff
@@ -208,7 +244,7 @@ public static async Task<UserCard[]> GetPlayerCardsAsync(string username, string
                     data = await Helper.DownloadPageAsync($"{Settings.SPLINTERLANDS_API_URL_FALLBACK}/cards/collection/{ username }?token={ accessToken }&username={ username }");
                 }
 
-                DateTime oneDayAgo = DateTime.Now.AddDays(-1);
+                DateTime oneDayAgo = DateTime.Now.AddDays(-2);
```

Comment:
onedayago now 2 days?

Reply:
yep, https://docs.splinterlands.com/cards/card-cooldown

Reply:
should be renamed to twoDaysAgo then probably?

Reply:
or maybe cooldown, in case the change it again